doc/test-framework/scorecard.md

```diff
@@ -177,6 +174,10 @@ Following the description of each internal [Plugin](#plugins). Note that are 8 i
 | Spec Fields With Descriptors | This test verifies that every field in the Custom Resources' spec sections have a corresponding descriptor listed in the CSV. This test has a maximum score equal to the total number of fields in the spec sections of each custom resource passed in via the `cr-manifest` option. |
 | Status Fields With Descriptors | This test verifies that every field in the Custom Resources' status sections have a corresponding descriptor listed in the CSV. This test has a maximum score equal to the total number of fields in the status sections of each custom resource passed in via the `cr-manifest` option. |
 
+## Exit Status
+
+In version v1alpha2, the scorecard return code is 1 if any of the tests executed did not pass and 0 if all selected tests pass.
```

Comment:
IHMO it still not fit in the doc. Note that if you add a section to explain the exit status you need say if you use v1alpha1 then .. however, if you are calling the scorecard with v1alpha2 then ...

Reply:
updated to include a comment about what happens in the v1alpha1 case.